docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+| Rate    | It splits into two `Int64Counter` counters with the names `metric_name.occurred` and `metric_name.total`. The `metric_name.occurred` counts only the number of non-zero occurencies, where the `metric_name.total` registers total number of positive and negative. We might reconsider it after the [xk6-output-opentelemetry#12](https://github.com/grafana/xk6-output-opentelemetry/issues/12). |
+| Trend   | `Float64Histogram`                                                                                                                                                                                                                                                                                                                                                                                 |
+
+## Build the k6 version
+
+To build a k6 binary with the extension, first, make sure you have [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/) installed on your machine.
+
+Then, open your terminal and run the following commands:
+
+```bash
+# Install xk6
+go install go.k6.io/xk6/cmd/xk6@latest
+
+# Build the k6 binary
+xk6 build --with github.com/grafana/xk6-output-opentelemetry
+
+... [INFO] Build environment ready
+... [INFO] Building k6
+... [INFO] Build complete: ./k6
+```
+
+xk6 will create the new k6 binary in the local folder.
+
+{{% admonition type="note" %}}
+
+To learn more about how to build custom k6 versions, check out [xk6](https://github.com/grafana/xk6).
+
+{{% /admonition %}}
+
+## Run the k6 test
+
+Using the k6 binary you built in the previous step, you can use the `--out xk6-opentelemetry` option when running your tests to use this extension:
+
+{{< code >}}
+
+```bash
+$ ./k6 run --out xk6-opentelemetry script.js
+```
+
+{{< /code >}}
+
+## Configuration
+
+The following options can be configured:
+
+| Name                               | Value                                                                                                                                                           |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `K6_OTEL_METRIC_PREFIX`            | Metric prefix. Default is empty.                                                                                                                                |
+| `K6_OTEL_FLUSH_INTERVAL`           | How frequently to flush metrics from k6 metrics engine. Default is `1s`.                                                                                        |
+| `K6_OTEL_EXPORT_INTERVAL`          | Configures the intervening time between metrics exports. Default is `1s`.                                                                                       |
```

Comment:
Related question, it sounds very low to be the default value, is it the default value in OTel world?

Reply:
By intuition, I would say `10s` could be a compromise, but I didn't conduct any advanced research to provide a rational number here.
What I know is, that storages like Prometheus, not originally designed for push-based architecture, work better with a more relaxed interval.
Furthermore, as it could be used by free-tiers then a more relaxed interval could be also convenient for Grafana Cloud. Because for paid plans, we mostly charge per active series and not per sample. So, if we send 1 series with 1k pushes we are not charging more than 1 series with 1 sample, instead, it costs more on our side because we need to process more sent data.

Reply:
Let's define it as `10s` then, I'm fine with that for now :+1: